_includes/downloads/opensearch-docker.markdown

```diff
@@ -4,7 +4,9 @@ The best way to try out OpenSearch is to use [Docker Compose](https://docs.docke
   - **macOS & Windows**: In Docker _Preferences_ > _Resources_, set RAM to at least 4 GB.
   - **Linux**: Ensure `vm.max_map_count` is set to at least 262144 as per the [documentation](/docs/opensearch/install/important-settings/).
 2. Download [docker-compose.yml](/samples/docker-compose.yml) into your desired directory
+  - **Note for OpenSearch 2.12 and Higher:**
+    When setting up the security demo configuration, set the initial admin password using the `OPENSEARCH_INITIAL_ADMIN_PASSWORD` environment variable. Create a `.env` file in the same folder as your `docker-compose.yml` and add the variable to `.env` file. Set the value to `OPENSEARCH_INITIAL_ADMIN_PASSWORD` variable to a strong admin password.  This configuration will be automatically utilized during the setup process.
```

Comment:
```suggestion
    When setting up the security demo configuration, set the initial admin password using the `OPENSEARCH_INITIAL_ADMIN_PASSWORD` environment variable. For more information, see [Setting up a demo configuration](https://opensearch.org/docs/latest//security/configuration/demo-configuration/).
```

Reply:
We have documentation about how to set up the `env` files for each distribution. Lets link to that.

Reply:
Here's the staging link for the linked page. The link in this PR won't work until we merge 2.12: https://1291fa08092a4a2dfd6d43f11d7bbcf3f22bf569.opensearch.org/docs/latest/security/configuration/demo-configuration/

Reply:
done